bluemira/base/config.py

```diff
@@ -94,6 +94,18 @@ class Configuration(ConfigurationSchema, ParameterFrame):
         ["C_Ejima", "Ejima constant", 0.4, "dimensionless", None, "Input (Ejima, et al., Volt-second analysis and consumption in Doublet III plasmas, Nuclear Fusion 22, 1313 (1982))"],
         ["m_s_limit", "Margin to vertical stability", 0.3, "dimensionless", None, "Input"],
         ["T_e_ped", "Electron temperature at the pedestal", 5.5, "keV", "Used in PLASMOD if fixed temperature pedestal model used", "Input"],
+        ["n_el_0", "Electron density on axis", 1.81e+20, "1/m^3", None, "Input"],
+        ["T_el_0", "Electron temperature on axis", 2.196e+01, "keV", None, "Input"],
+        ["rho_ped_n", "Density pedestal r/a location", 0.94, "dimensionless", None, "Input"],
+        ["rho_ped_t", "Temperature pedestal r/a location", 0.976 , "dimensionless", None, "Input"],
+        ["n_el_ped", "Electron density pedestal height", 1.086e+20, "1/m^3", None, "Input"],
+        ["T_el_ped", "Electron temperature pedestal height", 3.74, "keV", None, "Input"],
+        ["alpha_n", "Density profile factor", 1.15, "dimensionless", None, "Input"],
+        ["alpha_t", "Temperature profile index", 1.905, "dimensionless", None, "Input"],
+        ["t_beta", "Temperature profile index beta", 2, "dimensionless", None, "Input"],
+        ["n_el_sep", "Electron density at separatrix", 1.5515e+19, "1/m^3", None, "Input"],
+        ["T_el_sep", "Electron temperature at separatrix", 0.48, "keV", None, "Input"],
+        ["rminor", "Minor radius", 2.183, "m", None, "Input"],
```

Comment:
This for sure already exists

Reply:
I removed `T_el_ped` and used the existing `T_e_ped`, and I removed `rminor` and used `A` and `R_0`